js/css/custom.css

```diff
@@ -1,13 +1,20 @@
-.wtHolder {
-  width: auto !important;
-  height: auto !important;
-}
-
-.handsontable .jupyter-widgets {
-    width: 100%;
-    height: 100%;
-}
-
+/* handsontable layout*/
+
+/* These properties can be overridden with the Layout widget.
+ *
+ *  - overflow: hidden;
+ *   the first parent element with defined dimension and overflow: hidden is considered as the container for the spreadsheet
+ *
+ *  - height: 250px;
+ *   we used a fixed value in pixels for the the natural height because `auto` yields a zero-height table in the Jupyter notebook.
+ *
+ */
+.handsontable.jupyter-widgets {
+   overflow: hidden;
+   height: 250px;
+}
```

Comment:
Note: the natural height is set to some arbitrary pixel value, allowing handstontable to render itself. It is later over-written by the layout widget to `auto`.